### ISSUE TITLE
Conflict in file provider path

### ISSUE THREAD

**Issue by AlexVincent525** [Sh1d0w/multi_image_picker#141]
Hi there!
Today I just met a problem with file provider path, which this package's is conflict with [flutter_webview_plugin](https://github.com/fluttercommunity/flutter_webview_plugin), and the author is here [@charafau](https://github.com/charafau).

Here's log for the error:
```
Error:
    Attribute meta-data#android.support.FILE_PROVIDER_PATHS@resource value=(@xml/filepaths) from [:flutter_webview_plugin] AndroidManifest.xml:17:17-50
    is also present at [com.sangcomz:FishBun:0.9.0] AndroidManifest.xml:19:17-55 value=(@xml/provider_paths).
    Suggestion: add 'tools:replace="android:resource"' to <meta-data> element at AndroidManifest.xml to override.
```
FishBun is provided by this package, so I think maybe this is related.
I'm using multi_image_picker with 4.2.1, and flutter_webview_plugin with 0.3.5.

Hope this can be figured out and fixed ASAP!

**Comment by Sh1d0w** [comment]
I think you need to report that in the FushBun repo, there is not much I can do here.

Did you try to do what the error suggests, e.g. to add ""tools:replace="android:resource ?

**Comment by Sh1d0w** [comment]
Btw the fix is easy but have to be done in FishBun. Previously when I was using Matisse in this repo, there was similar problem because I was extending the default file provider.

Here is a reference how I swapped it with custom one to resolve conflicts https://github.com/Sh1d0w/multi_image_picker/commit/647fcd717741a545769a1e6203442ffb03ecd3d0

This however needs to be done in FushBun, because the file provider is hardcoded there not in my repo.

**Comment by AlexVincent525** [comment]
> I think you need to report that in the FushBun repo, there is not much I can do here.
> 
> Did you try to do what the error suggests, e.g. to add ""tools:replace="android:resource ?

That doesn't help this correct, but I'll bring this to FishBun.

**Comment by AlexVincent525** [comment]
Thank you and I'll update this if any update in FishBun.

**Comment by Sh1d0w** [comment]
FYI opened a PR in FishBun repo https://github.com/sangcomz/FishBun/pull/151

**Comment by Sh1d0w** [comment]
This should be fixed in version 4.3.1

**Comment by AlexVincent525** [comment]
Thx a lot.
